jolt-core/src/poly/one_hot_polynomial.rs

```diff
@@ -396,7 +432,13 @@ impl<F: JoltField> OneHotPolynomial<F> {
                     let min_row_index = chunk_index * chunk_size;
                     let max_row_index = min_row_index + chunk_size;
 
-                    for (t, k) in self.nonzero_indices.iter().enumerate() {
+                    for (t, k) in self
+                        .nonzero_indices
+                        .as_ref()
+                        .expect("nonzero_indices not initialized")
```

Comment:
if we replace expect with unwrap, does it stack into one line? Maybe it's better to have it as one line. Applicable everywhere, since expect shouldn't be triggered regardless.